app/uk/gov/hmrc/merchandiseinbaggage/controllers/PaymentCalculationController.scala

```diff
@@ -48,27 +49,32 @@ class PaymentCalculationController @Inject()(
           .fold(actionProvider.invalidRequestF(goodsDestinationUnansweredMessage)) { destination =>
             request.declarationJourney.declarationType match {
               case Import =>
-                for {
-                  paymentCalculations <- calculationService.paymentBECalculation(goods)
-                  rates               <- calculationService.getConversionRates(goods)
-                } yield {
-                  if (paymentCalculations.totalGbpValue.value > destination.threshold.value)
-                    Redirect(routes.GoodsOverThresholdController.onPageLoad())
-                  else
-                    Ok(
-                      view(
-                        paymentCalculations,
-                        rates,
-                        checkYourAnswersIfComplete(routes.CustomsAgentController.onPageLoad()),
-                        backButtonUrl))
-                }
+                calculationService
+                  .paymentBECalculation(goods)
```

Comment:
can we rename this to just be paymentCalculation now that the old implementation has been deleted?

Reply:
will rename it next when removing old code